packages/action-menu/test/index.ts

```diff
@@ -566,12 +566,16 @@ export const testActionMenu = (mode: 'sync' | 'async'): void => {
 
             expect(openSpy.callCount).to.equal(1);
         });
-        it('opens, then closes, on subsequent clicks', async () => {
+        it('opens, then closes, on subsequent clicks', async function () {
```

Comment:
Are we specifically testing that the action menu opens on pointer down? Because, to me, a click event feels ambiguous, like the menu could open on either pointerUp or pointerDown. 

I know that there's changes to `Picker.ts` that makes it pretty clear what we expect to happen on pointer up (to me), so why doesn't the Action Menu also need this clarification?

Reply:
The only _net new_ test is in Picker, but I can add a similar version to the other packages, too. Good call!